queries/nu/highlights.scm

```diff
@@ -286,6 +289,9 @@ key: (identifier) @property
 
 (long_flag_identifier) @attribute
```

Comment:
Thinking about it; isn't this redundant with line 282?

Reply:
Definitely, there's a lot to refactor.

Reply:
Then it would be appreciated, if you're up for it. (Not the whole query, just the `@attribute` patterns.)

If not, that's ok too, but then keep the new `"@"` pattern next to the other `@attribute` ones. (I am assuming that you're not relying on the ordering here to have this overwritten later.)